lib/galaxy/config/__init__.py

```diff
@@ -549,6 +549,12 @@ def _process_config(self, kwargs):
         self.database_engine_options = get_database_engine_options(kwargs)
         self.database_create_tables = string_as_bool(kwargs.get('database_create_tables', 'True'))
         self.database_encoding = kwargs.get('database_encoding')  # Create new databases with this encoding
+
+        # Database data tables connection
+        if not self.data_table_connection:  # Provide default if not supplied by user
```

Comment:
Can you reuse ``get_database_engine_options`` with like a data_tables prefix? This would allow the admin to configure a lot more of this.